### ISSUE TITLE
Improve conversion to data log in ephys formats

### ISSUE THREAD

**Issue by MichaelClerx** [myokit/myokit#984]
- ~ABF: Replace different methods with a single one with switches~
- Make sweep joining optional
- Use 0. etc naming for sweeps when not joined
- Use channel names in log
- Similar for ATF
- Similar for WCP

Full to-do in #985

**Comment by MichaelClerx** [comment]
Grepping for datalog vs data_log shows very little public API using either:

```
./myokit/tests/test_datablock.py:    def test_from_data_log(self):
./myokit/tests/test_formats_wcp.py:    def test_data_log_conversion(self):
[michael@localhost myokit]$ grep "datalog" ./myokit/ -Ir
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer.ico',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-16.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-24.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-32.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-48.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-64.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-96.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-128.xpm',
./myokit/gui/datalog_viewer.py:#        'icon-datalog-viewer-256.xpm',
./myokit/gui/datalog_viewer.py:            '.csv': self.load_datalog,
./myokit/gui/datalog_viewer.py:            '.zip': self.load_datalog,
./myokit/gui/datalog_viewer.py:    def load_datalog(self, filename):
./myokit/_bin/install-lin/myokit-datalog-viewer.desktop:#Icon=<?=icons?>/icon-datalog-viewer.png
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-1-empty.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-2-windows.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-3-old-mac.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-4-empty-lines.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-5-semicolons.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-6-open-string.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-7-empty-lines-2.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-8-unquoted-header.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-9-double-quoted-header.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-10-just-spaces.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-11-just-a-semicolon.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-12-bad-header.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-13-header-with-empty-1.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-14-header-with-empty-2.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-15-header-with-empty-3.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-16-wrong-columns-in-data.csv')
./myokit/tests/test_datalog.py:        path = os.path.join(DIR_IO, 'datalog-17-non-float-data.csv')
./myokit/tests/test_simulation_cvodes.py:        # Tested against offline method in test_datalog.py
./myokit/__init__.py:from ._datalog import (     # noqa
./myokit/__main__.py:    place_file(path, 'myokit-datalog-viewer.desktop', True)
./myokit/__main__.py:    import myokit.gui.datalog_viewer
./myokit/__main__.py:    myokit.gui.run(myokit.gui.datalog_viewer.DataLogViewer, *filenames)
```

Public:
- myokit.DataLog (two words)
- Module myokit.gui.datalog_viewer (one word)
- DataLogViewer (two words)
- Some filenames use datalog (one word)

**Comment by MichaelClerx** [comment]
Sometimes just called "log":
- DataBlock1d.from_log & to_log
- DataBlock2d.from_log & to_log
- myokit.prepare_log

**Comment by MichaelClerx** [comment]
Closed in #998
